Pear/Cells/EditProfileSectionTableViewCell.swift

```diff
@@ -93,29 +100,48 @@ class EditProfileSectionTableViewCell: UITableViewCell {
              make.top.equalTo(titleLabel.snp.bottom)
         }
     }
-
-    func configure(with interest: Interest) {
+    
+    func configure(with interest: Interest, user: UserV2, index: Int, delegate: didUpdateInterestsDelegate) {
+        self.cellType = "Interest"
         titleLabel.text = interest.name
         categoriesLabel.text = interest.subtitle
         if let interestImageUrl = URL(string: interest.imgUrl) {
             interestImageView.kf.setImage(with: interestImageUrl)
         }
+        self.interestsIndexDeleted = index
+        self.user = user
+        self.interestsDelegate = delegate
         setupConstraints()
     }
-
-    func configure(with group: Group) {
+    
+    func configure(with group: Group, user: UserV2, index: Int, delegate: didUpdateGroupsDelegate) {
+        self.cellType = "Group"
         titleLabel.text = group.name
         categoriesLabel.text = nil
         if let groupImageUrl = URL(string: group.imgUrl) {
             interestImageView.kf.setImage(with: groupImageUrl)
         }
+        self.groupsIndexDeleted = index
+        self.user = user
+        self.groupsDelegate = delegate
         setupConstraints()
     }
     
     
     @objc private func closeButtonPressed() {
-        //TODO: Add delete functionality
-        print("Close button")
+        // Remove the selected interest from the datasource and send this updated user back to the previous VC
+        // based on whether or not we're currently on the interests or groups VC.
+        if (self.cellType == "Interest") {
```

Comment:
To make the code look cleaner, you should only be using `self` when you really need it. For example, inside an escaping closure or when a class attribute and parameter have the same name